src/input/evdev.c

```diff
@@ -195,6 +207,49 @@ static char evdev_convert_value_byte(struct input_event *ev, struct input_device
   }
 }
 
+void *HandleMouseEmulation(void* param)
+{
+  struct input_device* dev = (struct input_device*) param;
+
+  while (dev->meThread) {
+    usleep(MOUSE_EMULATION_POLLING_INTERVAL);
+
+    short rawX;
+    short rawY;
+
+    float xParmsMax;
+    float yParmsMax;
+
+    // Determine which analog stick is currently receiving the strongest input
+    if ((uint32_t)abs(dev->leftStickX) + abs(dev->leftStickY) > (uint32_t)abs(dev->rightStickX) + abs(dev->rightStickY)) {
+      rawX = dev->leftStickX;
+      rawY = dev->leftStickY;
+      xParmsMax = (float)dev->xParms.max;
```

Comment:
These values are already normalized by `evdev_convert_value()` to -32768 to 32767 range, so you shouldn't need to normalize them again.